framework/vulkan_sample.cpp

```diff
@@ -85,9 +86,11 @@ bool VulkanSample::prepare(Platform &platform)
 
 	LOGI("Initializing Vulkan sample");
 
+	bool headless = platform.using_extension<::extensions::Headless>();
+
 	// Creating the vulkan instance
 	add_instance_extension(platform.get_surface_extension());
-	instance = std::make_unique<Instance>(get_name(), get_instance_extensions(), get_validation_layers(), is_headless(), api_version);
+	instance = std::make_unique<Instance>(get_name(), get_instance_extensions(), get_validation_layers(), headless);
```

Comment:
How is the api_version passed to the instance? Has this been moved to a different place?

Reply:
I think we have missed the [recent commit](https://github.com/KhronosGroup/Vulkan-Samples/commit/8f04afc7fea0044b5e2713de64ce87353b0073a7) that added this parameter, PRs need a rebase

Reply:
Rebased, but for some reason i didn't get a conflict on this line. I added the `api_version` as a quick fix but there may be other occurrences where this has happened and I have missed it